### ISSUE TITLE
Windows 10 navigator.mediaDevices.getUserMedia captures the wrong screen.

### ISSUE THREAD

**Issue by spicykoala** [electron/electron#11609]
* Electron version:
1.7.0+  (it works in 1.6.15)
* Operating system:
Windows 10 x64

### Expected behavior
When passing in a screen id getUserMedia should return a stream for the corresponding screen.

### Actual behavior
When passing in a screen id getUserMedia returns a stream from a different screen, or a broken stream in some situations.

** means primary monitor

Monitor configuration and results: 
- [ 1**, 2 ] - Capturing either screen works
- [ 1, 2** ] - Capturing screen 1 returns a stream for screen 2, capturing screen 2 returns a broken stream
- [ 2, 1** ] - Capturing screen 1 works, capturing screen 2 returns a broken stream.
- [ 2**, 1 ] - capturing screen 1 returns a stream for screen 2, capturing screen 2 returns stream for screen 1.

### How to reproduce

If you provide a URL, please list the commands required to clone/setup/run your repo e.g.
```bash
git clone https://github.com/hokein/electron-screen-recorder.git
cd electron-screen-recorder
npm install
npm uninstall electron-prebuilt --save-dev
npm install electron@1.7.10 --save
./node_modules/.bin/electron .
```
Attempt one of the broken configurations above to see results.

**Comment by spicykoala** [comment]
I just wanted to note that I am still experiencing this issue in 1.8.2 :(

**Comment by alexeykuzmin** [comment]
@spicykoala Can you please check 1.8.3?

**Comment by alexeykuzmin** [comment]
@zcbenz, @loc Can this one be closed since the #9933 is fixed?

**Comment by spicykoala** [comment]
Yes - this is fixed in 1.8.3 👍
